code/game/objects/items/weapons/shields.dm

```diff
@@ -41,13 +41,13 @@
 	. = ..()
 	switch(get_block_chance(user))
 		if(0 to 30)
-			to_chat(user, "So heavy... You feel doubtful in your ability to parry with this shield. Maybe if you changed your grip?")
+			to_chat(user, "So heavy... You doubt your ability to parry with this shield until you grow stronger.")
 		if(31 to 45)
-			to_chat(user, "Holding this feels a little clumsy.")
+			to_chat(user, "Holding this feels a little clumsy. Perhaps you should hit the gym.")
 		if(46 to 55)
-			to_chat(user, "With a comfortable grip, you feel condifent in your ability to parry with this shield.")
+			to_chat(user, "Still a little heavy, but you feel confident in your ability to parry with this shield.")
```

Comment:
This makes no sense, it's not like you see the previous messages when you have this robustness.

Reply:
Oh you mean the implication of the word "Still". Right, I get that now.